pegasus_it.ts

```diff
@@ -0,0 +1,548 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!DOCTYPE TS>
+<TS version="2.1" language="it_IT">
+<context>
+    <name>FilePicker</name>
+    <message>
+        <location filename="../src/frontend/menu/settings/gamedireditor/FilePicker.qml" line="81"/>
+        <source>Select a collection file below (metadata.pegasus.txt or metadata.txt).
+The directory of the file will then be added to the list.</source>
+        <translation>Seleziona un file collezione (metadata.pegasus.txt o metadata.txt).
+        La cartella del file sara' aggiunta alla lista.</translation>

```

Comment:
I think these leading spaces shouldn't be there.

Reply:
fixed